### PR TITLE
Feature/ng 10

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Update dependencies to ng 10

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -71,24 +76,53 @@
...
-            ],
-            "scripts": [],
+            "tsConfig": "tsconfig.spec.json",
+            "karmaConfig": "karma.conf.js",
             "assets": [
               "src/favicon.ico",
               "src/assets"
+            ],
+            "styles": [
+              "src/styles.scss"
```
Comment (joebochill) [inline angular.json:97]: where did the themes and typography references go?

**Review by joebochill — APPROVED** [review]

File: angular.json (hunk 1 above)
Comment (emclaug2) [reply angular.json:97]: They should be added there now, right after the src/styles.scss.

Comment (joebochill) [reply angular.json:97]: I'm not seeing them there. Just the src styles by itself on line 97.

image

Comment (emclaug2) [reply angular.json]: Ah you're right, this is missing the styles when we run our unit tests; these are added now.

**Pull request merged**